bootstrap.sh

```diff
@@ -254,136 +233,25 @@ xcode_license() {
 }
 
 install_homebrew() {
-  # Setup Homebrew directory and permissions.
   logn "Installing Homebrew:"
-  HOMEBREW_PREFIX="$(brew --prefix 2>/dev/null || true)"
-  HOMEBREW_REPOSITORY="$(brew --repository 2>/dev/null || true)"
-  if [ -z "$HOMEBREW_PREFIX" ] || [ -z "$HOMEBREW_REPOSITORY" ]; then
-    UNAME_MACHINE="$(/usr/bin/uname -m)"
-    if [[ "$UNAME_MACHINE" == "arm64" ]]; then
-      HOMEBREW_PREFIX="/opt/homebrew"
-      HOMEBREW_REPOSITORY="${HOMEBREW_PREFIX}"
-    else
-      HOMEBREW_PREFIX="/usr/local"
-      HOMEBREW_REPOSITORY="${HOMEBREW_PREFIX}/Homebrew"
-    fi
-  fi
-  [ -d "$HOMEBREW_PREFIX" ] || sudo_askpass mkdir -p "$HOMEBREW_PREFIX"
-  if [ "$HOMEBREW_PREFIX" = "/usr/local" ]; then
-    sudo_askpass chown "root:wheel" "$HOMEBREW_PREFIX" 2>/dev/null || true
-  fi
-  (
-    cd "$HOMEBREW_PREFIX"
-    sudo_askpass mkdir -p Cellar Caskroom Frameworks bin etc include lib opt sbin share var
-    sudo_askpass chown "$USER:admin" Cellar Caskroom Frameworks bin etc include lib opt sbin share var
-  )
-
-  [ -d "$HOMEBREW_REPOSITORY" ] || sudo_askpass mkdir -p "$HOMEBREW_REPOSITORY"
-  sudo_askpass chown -R "$USER:admin" "$HOMEBREW_REPOSITORY"
-
-  if [ $HOMEBREW_PREFIX != $HOMEBREW_REPOSITORY ]; then
-    ln -sf "$HOMEBREW_REPOSITORY/bin/brew" "$HOMEBREW_PREFIX/bin/brew"
-  fi
-
-  # Download Homebrew.
-  export GIT_DIR="$HOMEBREW_REPOSITORY/.git" GIT_WORK_TREE="$HOMEBREW_REPOSITORY"
-  git init $Q
-  git config remote.origin.url "https://github.com/Homebrew/brew"
-  git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-  git fetch $Q --tags --force
-  git reset $Q --hard origin/master
-  unset GIT_DIR GIT_WORK_TREE
-  logk
-
-  # Update Homebrew.
-  export PATH="$HOMEBREW_PREFIX/bin:$PATH"
-  logn "Updating Homebrew:"
-  [ -z "$QUICK" ] && brew update --quiet
+  sudo_askpass chown "$USER" /opt
+  where="/opt/homebrew"
+  [ -n "$CI" ] && where="/usr/local/Homebrew"
+  [ -d "$where" ] || git clone --depth=1 "https://github.com/Homebrew/brew" "$where"
+  export PATH="${where}/bin:${PATH}"
+  [ -z "$QUICK" ] && {
+    logn "Updating Homebrew"
+    brew update --quiet
+  }
   logk
 
-  # On Apple M1 machines we need to add this to the profile
-  if [[ "$(uname -m)" == "arm64" ]]; then
-    shell_profile=$(get_brew_profile)
-    eval "$(/opt/homebrew/bin/brew shellenv)"
-    if ! grep -qF "brew shellenv" "${shell_profile}"; then
-      #shellcheck disable=SC2016
-      echo -e 'eval "$(/opt/homebrew/bin/brew shellenv)"' >>${shell_profile}
-    fi
+  if ! grep -qF "brew shellenv" "${HOME}/.zshrc"; then
```

Comment:
Do we require folks to use zsh at sentry? Or should this be smarter and update bashrc and/or zshrc?